src/DataAggregator/NodeScopedWorkers/NodeMempoolFullTransactionReaderWorker.cs

```diff
@@ -86,6 +87,12 @@ namespace DataAggregator.NodeScopedWorkers;
 /// </summary>
 public class NodeMempoolFullTransactionReaderWorker : NodeWorker
 {
+    private static readonly IDelayBetweenLoopsStrategy _delayBetweenLoopsStrategy =
+        new ExponentialBackoffDelayBetweenLoopsStrategy(
+            TimeSpan.FromMilliseconds(500),
+            TimeSpan.FromMilliseconds(500),
+            1, 2, 5);
```

Comment:
I might be tempted to add in the optional prefixes to variable names to explain what these numbers mean :)

EG:
```
new ExponentialBackoffDelayBetweenLoopsStrategy(
    minDelayAfterSuccess: X,
    ...
)
```

Reply:
Rider IDE adds it automatically (which is a bit annoying). But yeah, named parameters wouldn't hurt here for anyone using a different IDE.